src/bun.js/webcore/blob.zig

```diff
@@ -1110,7 +1116,10 @@ pub const Blob = struct {
                     break :brk result;
                 },
                 .err => |err| {
-                    return JSC.JSPromise.rejectedPromiseValue(globalThis, err.toJSC(globalThis));
+                    return JSC.JSPromise.rejectedPromiseValue(
```

Comment:
These need to check if it's a file descriptor first. This will crash if pathlike's active union member is `fd`

Reply:
Fixed.